binaries/daemon/src/lib.rs

```diff
@@ -185,6 +185,18 @@ impl Daemon {
         uv: bool,
         log_destination: LogDestination,
     ) -> eyre::Result<DataflowResult> {
+        // Spawn local listener for dynamic nodes
+        let (events_tx, events_rx) = flume::bounded(10);
+        let _listen_port = local_listener::spawn_listener_loop(
+            (LOCALHOST, DORA_DAEMON_LOCAL_LISTEN_PORT_DEFAULT).into(),
+            events_tx,
+        )
+        .await?;
```

Comment:
This means that `dora run` will now fail if there is any `dora daemon` or other `dora run` running concurrently. I'm not sure if this limitation is worth it...

Maybe we can choose a free port instead and make the port number used in `init_from_node_id` overridable (e.g. through an env variable)? Then we could make `dora run` print the port number to the terminal and the user could set the env variable accordingly.